api/src/main/java/org/openmrs/module/rwandahivflowsheet/impl/pih/ImageMapping.java

```diff
@@ -185,7 +185,7 @@ public String getImageTypeString(){
 				return o.getConcept().getName().getName();
 			} else if (o.getValueCoded() != null)
 				try {
-					return o.getValueCoded().getBestName(Context.getLocale()).getName();
```

Comment:
I have been using o.getValueCoded().getDisplayString(), which might be a better choice.  Need to look at how they differ.  The main thing is you don't want this to ever return null or an empty string, which I think the getDisplayString() guards against.

Reply:
This comment applies to all of the other places where displaying a concept is done this way in this PR

Reply:
[Done](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/4048a80bbb356519bdd8a10b0ec226a093d4de74)